invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -52,16 +45,17 @@ def get_name_identifiers(self, obj):
         for identifier in identifiers:
             scheme = identifier["scheme"]
             value = identifier["identifier"]
-            uri = self.URIS.get(scheme)
 
             name_id = {
                 "nameIdentifier": value,
                 "nameIdentifierScheme": scheme.upper(),
             }
 
+            uri = to_url(value, scheme)
             if uri:
-                name_id["nameIdentifier"] = uri + value
-                name_id["schemeURI"] = uri
+                name_id["nameIdentifier"] = uri
+                name_id["schemeURI"] = LANDING_URLS.get(scheme).format(
```

Comment:
I don't think we can do it like this as they are for the landing pages and might not be the same as the Scheme's URI or the scheme might not have a URI. As long as we include the ``nameIdentifierScheme`` we can remove the ``schemeURI``.

Reply:
:+1: Removing